packages/core/src/Practical.tsx

```diff
@@ -60,24 +69,38 @@ interface PracticalProviderProps extends SimpleToastsDurations {
    *
    * @default Toasts are centered at the top and always visible.
    */
-  readonly toastsOptions?: ToastsOptions
+  readonly toastsOptions?: UserToastsOptions
+}
+
+const mergeToastOptions = (
+  defaultOptions: ToastsOptions,
+  userOptions?: UserToastsOptions
+) => {
+  const toastOptions = {
+    ...defaultOptions,
+    ...userOptions,
+  }
+
+  return toastOptions
 }
```

Comment:
Do this inline instead, it is just one line.

Reply:
Done